magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -83,29 +83,43 @@ export default class DatasetPageSuggestForm extends React.Component {
                 maxHeight: "95vh"
             }
         };
+        let contactPointMatch;
+        if (typeof this.props?.contactPoint === "string") {
+            const emailRegex = /[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}/gi;
```

Comment:
That's my bad here. I think we probably can remove the `g` modifier so it only searches for the first occurrence. This change doesn't impact other code.